pkg/api/validation/validation.go

```diff
@@ -119,6 +119,16 @@ func validateGCEPersistentDisk(PD *api.GCEPersistentDisk) errs.ValidationErrorLi
 	return allErrs
 }
 
+func ValidatePersistentVolume(persistentvolume *api.PersistentVolume) errs.ValidationErrorList {
+	allErrs := errs.ValidationErrorList{}
+	return allErrs
```

Comment:
Yes, this will require validation, of course.  First pass through for a proof-of-concept will be rough, to say the least.